packages/protocol/src/actions.ts

```diff
@@ -1,5 +1,7 @@
+export type CustomFieldType = 'START' | 'TASK' | 'CASE';
+
 export interface InscriptionActionHandlers {
   newHtmlDialog: undefined;
   newProcess: undefined;
-  openFile: string;
+  openFile: `pageUri=${string}` | `customField=${CustomFieldType},${string}` | `doc=${string}`;
```

Comment:
this one is also asking for a clear structure as it seems ;-)

Reply:
I can provide a bit of type safety here, but this should definitely also be handled by a clear type from the protocol